docs/data/base/getting-started/quickstart/quickstart.md

```diff
@@ -0,0 +1,244 @@
+# Quickstart
+
+<p class="description">Get started with Base UI, a library of headless ("unstyled") React UI components and low-level hooks.</p>
+
+## Installation
+
+`@mui/base` is completely standalone – run one of the following commands to add Base UI to your React project:
+
+### With npm
+
+```sh
+npm install @mui/base
+```
+
+### With yarn
+
+```sh
+yarn add @mui/base
+```
+
+### With pnpm
+
+```sh
+pnpm add @mui/base
+```
+
+### Peer dependencies
+
+<!-- #react-peer-version -->
+
+[`react`](https://www.npmjs.com/package/react) >= 17.0.0 and [`react-dom`](https://www.npmjs.com/package/react-dom) >= 17.0.0 are peer dependencies.
+
+## Implementing a Button
```

Comment:
I wonder if it is worth adding a paragraph that explains why you should use Base UI button instead of a plain HTML button.

I guess a lot of people might have this question in their minds. cc @samuelsycamore

Reply:
Great idea @siriwatknp !